plugins/org.jkiss.dbeaver.ext.gaussdb.ui/src/org/jkiss/dbeaver/ext/gaussdb/ui/GaussDBCreateDatabaseDialog.java

```diff
@@ -0,0 +1,224 @@
+package org.jkiss.dbeaver.ext.gaussdb.ui;
```

Comment:
Please add copyrights for all java files. You can get the copyright from other files.